qemu/tests/driver_in_use.py

```diff
@@ -119,8 +119,12 @@ def run_subtest(sub_type):
         if stress_thread:
             stress_thread.join(timeout=timeout,
                                suppress_exception=suppress_exception)
+        if sub_type != "shutdown":
+            run_subtest(bg_stress_test)
```

Comment:
it's better to use a name more clear

Reply:
and I think it's not good to fillter a special case

Reply:
Updated,thanks.
1.rename "sub_type" to "main_test" to distinguish it from background test and make it more clear to know what test are going to run,such as:
 run_subtest(main_test)
 run_subtest(bg_stress_test)
2.use vm.is_alive() instead of filtering shutdown case to decide whether run bg_stress_test after main test.

Reply:
job pass after patch udpate:
(1/3) smp_8.2048m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.viorng_in_use.with_live_migration.before_bg_test: PASS (518.35 s)
(2/3) smp_8.2048m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.viorng_in_use.with_live_migration.during_bg_test: PASS (399.87 s)
(3/3) smp_8.2048m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.viorng_in_use.with_live_migration.after_bg_test: PASS (223.87 s)
(1/3) smp_8.2048m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.viorng_in_use.with_shutdown.before_bg_test: PASS (494.08 s)
(2/3) smp_8.2048m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.viorng_in_use.with_shutdown.during_bg_test: PASS (269.48 s)
(3/3) smp_8.2048m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.viorng_in_use.with_shutdown.after_bg_test: PASS (455.96 s)